### PR TITLE
Validation: Added Missing Column Headers

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing column headers.

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/smoothstreamingerroreventargs-methods-microsoft-web-media-smoothstreaming.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: SmoothStreamingErrorEventArgs Methods (Microsoft.Web.Media.SmoothStreaming)
+title: Smooth Streaming Error Event Args Methods
+description: Describes the methods for smooth streaming errors, and provides various methods and references to reference.
```
Comment (kturetz) [inline iis/extensions/smooth-streaming-client/smoothstreamingerroreventargs-methods-microsoft-web-media-smoothstreaming.md:3]: "and references to consult for additional information."

File: iis/extensions/transform-manager/ilogger-methods-microsoft-web-media-transformmanager.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: ILogger Methods (Microsoft.Web.Media.TransformManager)
+description: Describes the ILogger methods and provides a table that outlines the name and description for various methods.
```
Comment (kturetz) [inline iis/extensions/transform-manager/ilogger-methods-microsoft-web-media-transformmanager.md:3]: "names and descriptions"

File: iis/extensions/transform-manager/taskdefinition-fields-microsoft-web-media-transformmanager.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TaskDefinition Fields (Microsoft.Web.Media.TransformManager)
+description: Describes the task definition type and provides a table that outlines the field, name, and description for various fields.
```
Comment (kturetz) [inline iis/extensions/transform-manager/taskdefinition-fields-microsoft-web-media-transformmanager.md:3]: Capitalize "Field, Name, and Description" if these are labels.

**Comment by dereklh7** [comment]
@Rick-Anderson @mcleblanc 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**